src/channel.cr

```diff
@@ -1,5 +1,19 @@
 require "fiber"
 
+# Channels enables to execute multiple tasks concurrently.
```

Comment:
No, channels are for communicating between fibers. They don't execute multiple tasks concurrently, that's what fibers do.